packages/@aws-cdk/aws-ssm/lib/parameter.ts

```diff
@@ -145,8 +145,35 @@ abstract class ParameterBase extends Resource implements IParameter {
 }
 
 const STRING_PARAM_TYPE = 'String';
+const SECURE_STRING_PARAM_TYPE = 'SecureString';
 const STRINGLIST_PARAM_TYPE = 'StringList';
 
+export interface ParameterAttributes {
```

Comment:
Should be called `StringParameterAttributes` to be consistent with `SecureStringParameterAttributes`.

Reply:
done